### PR TITLE
Added preauthorize annotations in Car controller

### PR DESCRIPTION
**Pull request by ArtemReizer**

### CODE REVIEW

File: src/main/java/com/jvavateam/carsharingapp/controller/CarController.java [hunk 1]
```diff
@@ -28,6 +29,7 @@ public class CarController {
     private final CarService carService;
 
     @PostMapping
+    @PreAuthorize("hasRole('ROLE_MANAGER')")
```
Comment (holsoni) [inline src/main/java/com/jvavateam/carsharingapp/controller/CarController.java]:
```suggestion
    @PreAuthorize("hasRole('MANAGER')")
```

**Review by holsoni — CHANGES_REQUESTED** [review]

**Review by holsoni — APPROVED** [review]
Good

File: src/main/java/com/jvavateam/carsharingapp/controller/CarController.java [hunk 2]
```diff
@@ -29,7 +29,7 @@ public class CarController {
     private final CarService carService;
 
     @PostMapping
-    @PreAuthorize("hasRole('ROLE_MANAGER')")
+    @PreAuthorize("hasRole('MANAGER')")
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/controller/CarController.java]: @PreAuthorize("hasAuthority('MANAGER')")

File: src/main/java/com/jvavateam/carsharingapp/controller/CarController.java [hunk 3]
```diff
@@ -38,20 +38,22 @@ public CarDtoResponse create(@Valid @RequestBody CarDtoRequest carDto) {
     }
 
     @GetMapping(value = "/{id}")
+    @PreAuthorize("hasAnyRole('MANAGER', 'CUSTOMER')")
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/controller/CarController.java]: @PreAuthorize("hasAnyAuthority('MANAGER', 'CUSTOMER')")

File: src/main/java/com/jvavateam/carsharingapp/controller/CarController.java [hunk 4]
```diff
@@ -38,20 +38,22 @@ public CarDtoResponse create(@Valid @RequestBody CarDtoRequest carDto) {
...
     @Operation(summary = "Get car by id",
             description = "Get certain car by id")
     public CarDtoResponse get(@PathVariable Long id) {
         return carService.getById(id);
     }
 
     @GetMapping
+    @PreAuthorize("hasAnyRole('MANAGER', 'CUSTOMER')")
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/controller/CarController.java]: @PreAuthorize("hasAnyAuthority('MANAGER', 'CUSTOMER')")

File: src/main/java/com/jvavateam/carsharingapp/controller/CarController.java [hunk 5]
```diff
@@ -38,20 +38,22 @@ public CarDtoResponse create(@Valid @RequestBody CarDtoRequest carDto) {
...
     @Operation(summary = "Get all cars")
     public List<CarDtoResponse> getAll() {
         return carService.getAll();
     }
 
     @PutMapping("/{id}")
-    @PreAuthorize("hasRole('ROLE_MANAGER')")
+    @PreAuthorize("hasRole('MANAGER')")
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/controller/CarController.java]: @PreAuthorize("hasAuthority('MANAGER')")

File: src/main/java/com/jvavateam/carsharingapp/controller/CarController.java [hunk 6]
```diff
@@ -60,7 +62,7 @@ public CarDtoResponse update(@RequestBody CarDtoRequest carDto,
     }
 
     @DeleteMapping("/{id}")
-    @PreAuthorize("hasRole('ROLE_MANAGER')")
+    @PreAuthorize("hasRole('MANAGER')")
```
Comment (IvanFadieiev) [inline src/main/java/com/jvavateam/carsharingapp/controller/CarController.java]: @PreAuthorize("hasAuthority('MANAGER')")

**Review by IvanFadieiev — CHANGES_REQUESTED** [review]
Good job! But hasRole is not working here

**Review by IvanFadieiev — APPROVED** [review]
Nice! :)

**Pull request merged**